boards/common/esp32/Kconfig

```diff
@@ -10,3 +10,13 @@ config BOARD_COMMON_ESP32
     select HAS_PERIPH_GPIO_IRQ
     select HAS_PERIPH_UART
     select HAS_PERIPH_UART_MODECFG
+
+    select HAVE_SAUL_GPIO
+
+config MODULE_BOARDS_COMMON_ESP32
```

Comment:
Why is this one selectable while the that one in `boards/common/esp8266/Kconfig` is not? The structure of `Makefile.dep` is the same for both.

Reply:
What do you mean by selectable? It has no prompt as well, the user can't select them, both are hidden symbols

Reply:
No particular reason, I probably did that on two separate days. Being a hidden symbol that is always enabled both approaches are pretty much the same. I changed to the `default y` to use less `select`. 190e76b439b98e89562b8454ec161a02beec55c0